app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherAdapter.kt

```diff
@@ -84,6 +94,18 @@ class TabSwitcherAdapter(private val context: Context, private val itemClickList
         val favicon: ImageView,
         val title: TextView,
         val url: TextView,
-        val close: ImageView
+        val close: ImageView,
+        val tabUnread: View
     ) : RecyclerView.ViewHolder(root)
+
+    companion object {
+
+        @DrawableRes val backgroundSelected = R.drawable.tab_background_selected
+        @DrawableRes val backgroundDefault = R.drawable.tab_background
+
+        val alphaSelected = 1.0f
```

Comment:
should the 4 variables in the companion object be styled as static constants? 

eg: `private const val ALPHA_SELECTED = 1.0f`

also, looks like this file needs its code formatted.